core/src/main/scala/dagr/core/execsystem/ResourceSet.scala

```diff
@@ -59,8 +59,8 @@ case class ResourceSet(cores: Cores = Cores.none, memory: Memory = Memory.none)
     * number of cores. Will greedily assign the highest number of cores possible.
     */
   def subset(minCores: Cores, maxCores: Cores, memory: Memory) : Option[ResourceSet] = {
-    val min = minCores.value
-    val max = maxCores.value
+    val min = math.ceil(minCores.value).toInt
+    val max = math.floor(maxCores.value).toInt
```

Comment:
I am curious about this change and why it was done.  I think it is perfectly valid to have `0.9` cores available, when a task requires `0.1` cores.